### PR TITLE
Install less module for strict hostonly mode

### PR DESCRIPTION
**Pull request by ryncsn**
This pull request changes...

## Changes
Install less module for strict hostonly mode. Strict hostonly mode is used by kdump, which expect the initramfs to be as small as possible. 

## Checklist
- [ ] I have tested it locally
- [ ] I have reviewed and updated any documentation if relevant
- [ ] I am providing new code and test(s) for it

Fixes #

### CODE REVIEW

File: modules.d/90kernel-modules/module-setup.sh [hunk 1]
```diff
@@ -19,13 +20,19 @@ installkernel() {
         IFS=$_OLDIFS
     }
 
-    is_block_dev() {
-        [ -e /sys/dev/block/$1 ] && return 0
+    record_block_dev_drv() {
+        for _mod in $(get_dev_module /dev/block/$1); do
+            [[ " $_hostonly_drvs " != *$_mod* ]] && _hostonly_drvs+=" $_mod"
+        done
+        [[ "$_hostonly_drve" ]] && return 0
```
Comment (haraldh) [inline modules.d/90kernel-modules/module-setup.sh:27]: typo! `_hostonly_drve` vs `_hostonly_drvs`

File: modules.d/90kernel-modules/module-setup.sh [hunk 2]
```diff
@@ -133,7 +133,6 @@ installkernel() {
         [[ $arch == aarch64 ]] && arch=arm64
         instmods "=crypto" "=arch/$arch/crypto" "=drivers/crypto"
     fi
-    :
```
Comment (haraldh) [inline modules.d/90kernel-modules/module-setup.sh:119]: commit c9e5b75308d858dfb865d3db0a7bad1d6062348d
Author: Harald Hoyer <harald@redhat.com>
Date:   Mon Mar 18 09:29:08 2013 +0100

    kernel-modules/module-setup.sh: don't fail hard on a missing module

**Review by haraldh — CHANGES_REQUESTED** [review]

File: modules.d/90kernel-modules/module-setup.sh (hunk 1 above)
Comment (ryncsn) [reply modules.d/90kernel-modules/module-setup.sh:27]: Thanks! I found this issue when testing but forgot to patch it up... Sorry, my bad.

File: modules.d/90kernel-modules/module-setup.sh (hunk 2 above)
Comment (ryncsn) [reply modules.d/90kernel-modules/module-setup.sh:119]: Ah, I got it. Thanks, I won't touch this line then.

**Pull request merged**